doc/visual-programming/source/widgets/evaluate/calibrationplot.md

```diff
@@ -7,21 +7,41 @@ Shows the match between classifiers' probability predictions and actual class pr
+   - *true and false positive rate*: displays the relationship between [TP and FP rate](https://en.wikipedia.org/wiki/False_positives_and_false_negatives) at changing probability thresholds. Threshold can be set by dragging the vertical line left or right.
+4. If *Apply Automatically* is ticked, changes are communicated automatically. Alternatively, click *Apply*.
+
+Examples
+--------
+
+At the moment, the only widget which gives the right type of signal needed by the **Calibration Plot** is [Test and Score](../evaluate/testandscore.md). The Calibration Plot will hence always follow Test and Score.
+
+Here is a typical example on the iris data, where we compare two classifiers (namely [Logistic Regression](../model/logisticregression.md) and [Random Forest](../model/randomforest.md)) and input them into [Test and Score](../evaluate/testandscore.md). Test and Score displays evaluation results for each classifier. Then we draw **Calibration Plot** to further analyze the performance of the classifiers. **Calibration Plot** enables you to see prediction accuracy of class probabilities in a plot.
+
+The widget is set to optimize F1 score. The user can drag the vertical black line left or right to set the probability threshold for the select target value. The information on the calibrated classifier are displayed in the info box on the left.
 
-Example
--------
+![](images/Calibration-Plot-Example1.png)
 
-At the moment, the only widget which gives the right type of signal needed by the **Calibration Plot** is [Test & Score](../evaluate/testandscore.md). The Calibration Plot will hence always follow Test & Score and, since it has no outputs, no other widgets follow it.
+In the second example, we show how to use the widget to output a calibrated model. We use [Data Sampler](../data/datasampler.md) to split the data into training and test subsets. We pass both the training and test subsets to **Test and Score** and train a [Logistic Regression](../model/logisticregression.md) model, which we pass to **Calibration Plot**. Note that only a single calibrated model can be on the output, hence the user must select a single model from the classifier list.
 
-Here is a typical example, where we compare three classifiers (namely [Naive Bayes](../model/naivebayes.md), [Tree](../model/tree.md) and [Constant](../model/constant.md)) and input them into [Test & Score](../evaluate/testandscore.md). We used the *Titanic* dataset. Test & Score then displays evaluation results for each classifier. Then we draw **Calibration Plot** and [ROC Analysis](../evaluate/rocanalysis.md) widgets from Test & Score to further analyze the performance of classifiers. **Calibration Plot** enables you to see prediction accuracy of class probabilities in a plot.
+Once the model is calibrated, we can pass it to [Predictions](../evaluate/predictions.md) and use it on training data.
```

Comment:
Why on training data? If can be used on any data...

Reply:
It is just an example, not an instruction.